src/app/core.py

```diff
@@ -0,0 +1,468 @@
+"""This file contains the core logic used in the SPDX Online Tools' APP and API"""
+
+import jpype
+import os
+
+from django.utils.datastructures import MultiValueDictKeyError
+from django.core.files.storage import FileSystemStorage
+from django.conf import settings
+
+from json import dumps, loads
+from time import time
+from traceback import format_exc
+from urllib.parse import urljoin
+
+import app.utils as utils
+
+
+def initialise_jpype():
+    """Start JVM if not already started, attach a Thread and start processing the request"""
+    
+    # Check is the JVM is already running or not. If not, start JVM.
+    if not jpype.isJVMStarted():
+        classpath = settings.JAR_ABSOLUTE_PATH
+        jpype.startJVM(jpype.getDefaultJVMPath(), "-ea", "-Djava.class.path=%s"%classpath)
+            
+    # Attach a thread to JVM and start processing
+    jpype.attachThreadToJVM()
+
+
+def license_compare_helper(request):
+    """
```

Comment:
Just confirming that you copied this code from app/views.py, right? Coz similar api code was outdated.

Reply:
Yeah, it's the code from app views.